pandas/core/tools/numeric.py

```diff
@@ -142,6 +147,10 @@ def to_numeric(arg, errors="raise", downcast=None):
     else:
         values = arg
 
+    if is_extension_array_dtype(arg) and isinstance(values, NumericArray):
+        is_numeric_extension_dtype = True
+        mask, values = values._mask, values._data
```

Comment:
you likely need to just take the masked values only for the following block of code

Reply:
Done